test.js

```diff
@@ -92,6 +92,24 @@ describe('store', function () {
     store.has('a.b.c.d.x').should.be.true;
   });
 
+  it('should return true if a key `.hasOwn()` on the store', function () {
+    store = new Store('eee');
+    store.set('ggg', 'fff');
+    store.set('abc', null);
+    store.hasOwn('ggg').should.be.true;
+    store.hasOwn('abc').should.be.true;
+    store.hasOwn('foo').should.be.false;
+  });
+
+  it('should return true if a nested key `.hasOwn()` on the store', function () {
```

Comment:
`.hasOwn()` is a typo in the description. it should be something like: "should return true if a nested key exists on the store"